rental_product_variant/views/product_view.xml

```diff
@@ -57,11 +57,9 @@
                     </button>
                 </div>
                 <xpath expr="//field[@name='categ_id']" position="replace" />
-                <xpath expr="//field[@name='type']" position="after">
```

Comment:
to keep the origin requirement, you can change it into "detailed_type"

Reply:
done, much better